### DISCUSSION TITLE
可否建立一个中文论坛感觉会更加方便

### DISCUSSION THREAD

**Discussion by SH-alaways** [zotero-chinese/wiki#98] [社区建设]
模板可以用https://discuss.flarum.org.cn/的模板
让大家在其中讨论zotero的使用 也可以邀请插件作者入驻 还有使用zotero比较厉害的大佬入驻

**Comment by northword** [comment]
不太好实现，我们没有很多的经费负担一个服务器。

且在国内开办论坛需要企业备案，不备案随时会被封禁，风险较大。

**Comment by SH-alaways** [comment]
可惜了，没有国内的论坛，zotero的信息都是零散的~

**Comment by northword** [comment]
> 可惜了，没有国内的论坛，zotero的信息都是零散的~

我们在努力整合资源，降低信息差。

我们有考虑 GitHub Discussion 做反代后作为论坛用，也考虑过 Gitee 的讨论区等等，但是还没有一个短板较小的方案，如果你有好的想法，也可以讲一讲~

**Comment by Yanc14** [comment]
插眼

**Comment by ZnqbuZ** [comment]
百度贴吧？不确定是否大家信任百度．另外我可以提供/维护至多 3C4.5G 的海外服务器，美国的国内各地 ping 平均大约 200ms，共同承担费用，免费也行

**Comment by SH-alaways** [comment]
> > 可惜了，没有国内的论坛，zotero的信息都是零散的~
> 
> 我们在努力整合资源，降低信息差。
> 
> 我们有考虑 GitHub Discussion 做反代后作为论坛用，也考虑过 Gitee 的讨论区等等，但是还没有一个短板较小的方案，如果你有好的想法，也可以讲一讲~

最好的方法就是建立一个自己国内的论坛，我看这个模板的论坛还不少，都需要备案吗。。

**Comment by northword** [comment]
> 最好的方法就是建立一个自己国内的论坛，我看这个模板的论坛还不少，都需要备案吗。。

只要服务器位于国内就需要备案，境外服务器开办论坛也会随时被 ban。

> 百度贴吧？不确定是否大家信任百度

Gitee Issue 也许也可以，似乎百度贴吧在一定程度上和 Gitee 能做的差不太多，Gitee 上其实是有一个讨论区的，只是并没有很多人在活跃，当然也可能与平台有关。（我没有详细对比，上一次上贴吧可能都是七八年前的事情了，如果有不恰当的地方还请指出）

> 另外我可以提供/维护至多 3C4.5G 的海外服务器，美国的国内各地 ping 平均大约 200ms，共同承担费用，免费也行

感谢！我们再讨论一下。

---

不过话说回来，Zotero 官方是有自己的论坛的，并且官方论坛并不禁止使用其他语言，相反，他们明确表示用户可以使用自己的母语表达。
此外，我们不清楚 Zotero 官方对于建立非官方中文论坛的态度，毕竟官方论坛是其开发者与用户交流的唯一渠道。并且 Zotero 官方也曾表示过不允许我们使用 Zotero 的商标。
第三点，我也担忧建立论坛后的活跃程度。说实话，论坛确实已经不是很多人经常接触的东西了。
（个人看法，不代表社区）

**Comment by ZnqbuZ** [comment]
> > 最好的方法就是建立一个自己国内的论坛，我看这个模板的论坛还不少，都需要备案吗。。
> 
> 只要服务器位于国内就需要备案，境外服务器开办论坛也会随时被 ban。

是，准确地说云服务商会直接封禁未备案主机的 80/443 端口．另外国内似乎已经在实际上禁止了个人备案论坛类网站．墙多半可以用 cloudflare 绕过？如果不行的话终极办法是用静态网页提供跳转链接．

另外未备案网站在国内搜索引擎那是三等公民．

> > 百度贴吧？不确定是否大家信任百度
> 
> Gitee Issue 也许也可以，似乎百度贴吧在一定程度上和 Gitee 能做的差不太多，Gitee 上其实是有一个讨论区的，只是并没有很多人在活跃，当然也可能与平台有关。（我没有详细对比，上一次上贴吧可能都是七八年前的事情了，如果有不恰当的地方还请指出）
> 
> 不过话说回来，Zotero 官方是有自己的论坛的，并且官方论坛并不禁止使用其他语言，相反，他们明确表示用户可以使用自己的母语表达。 此外，我们不清楚 Zotero 官方对于建立非官方中文论坛的态度，毕竟官方论坛是其开发者与用户交流的唯一渠道。并且 Zotero 官方也曾表示过不允许我们使用 Zotero 的商标。 第三点，我也担忧建立论坛后的活跃程度。说实话，论坛确实已经不是很多人经常接触的东西了。 （个人看法，不代表社区）

我也不用贴吧，提的原因是我使用的科学计算软件 Mathematica 在贴吧提供在我看来比较优秀的非官方支持（甚至非法支持，他们提供软件破解方法）——依赖于诲人不倦的管理员（团队）．此外 Gitee Issue 相比于贴吧在百度的 SEO 应该还是要劣势不少．

**Comment by xie-dd** [comment]
1. 我觉得 QQ频道 挺方便的，另外 Github Discussion 也可以考虑。
2. 非常不建议使用Gitee【除非gitee能解决只能搜索标题，不能搜索内容的问题】
